pkg/hs/urbit-king/lib/Urbit/King/CLI.hs

```diff
@@ -221,6 +222,13 @@ opts = do
       <> help "Ames port"
       <> hidden
 
+    oAmesLatency <-
+      optional
+      $  option (map (*1000) auto)
+      $  metavar "DELAY"
+      <> long "ames-latency"
+      <> help "Milliseconds before sending ames event"
+
```

Comment:
You probably want `<> hidden` here so that this option doesn't show up on the brief line.

Reply:
I wouldn't do the `(* 1000)` here. Command-Line arguments data structure should closely map to actual user input. Do the conversion later, unless it's too annoying. Right now the code isn't setup for this (there is no function to convert from CLI parameters to runtime options. This already hurts in a few places now, so it's ripe for doing).

This is probably fine, just commenting here b/c there's a better way to do this.